src/clj/puppetlabs/services/legacy_routes/legacy_routes_core.clj

```diff
@@ -46,21 +85,18 @@
   [handler mount-point api-version request]
   (let [{{environment :environment} :params
          uri :uri
-         query-string               :query-string} request
-        path-info (str "/" (-> (string/split uri #"/" 3) (nth 2)))
-        query-params (if (str/blank? query-string)
-                       {}
-                       (ring-codec/form-decode query-string))]
+         query-string :query-string} request
+        path-info (str "/" (-> (string/split uri #"/" 3) (nth 2)))]
```

Comment:
I guess this isn't new for this PR, but I'd prefer to not be doing our own hand-rolled URI parsing like this.  Could we instead construct a `java.net.URI` with `uri` as the argument and then call `.getRawPath` on that object to obtain the path?